app/store/chatList.go

```diff
@@ -230,61 +244,112 @@ func UpdateTimestamps() {
 			}
 			for _, m := range s.Messages {
 				m.HTime = helpers.HumanizeTimestamp(m.SentAt)
-				//qml.Changed(m, &m.HTime)
 			}
 			s.When = s.Messages[len(s.Messages)-1].HTime
-			//qml.Changed(s, &s.When)
 		}
 	}
 }
-func (s *Sessions) Get(tel string) *Session {
+
+// Get returns the session by it's id
+func (s *Sessions) Get(id int64) (*Session, error) {
+	for _, ses := range s.Sess {
+		if ses.ID == id {
+			return ses, nil
+		}
+	}
+	return nil, fmt.Errorf("Session with id %d not found", id)
+}
+
+// GetByE164 returns the session by the telephone number amd creates it if it doesn't exists
```

Comment:
I don't see anything which create the session here, is that comment up-to-date?

Reply:
This is not for creating a session but to retrieve the an existing session.

Reply:
You were right, it's solved now.